src/test/enquireBusInfo.test.js

```diff
@@ -0,0 +1,22 @@
+import chai, { expect } from 'chai';
+import chaiHttp from "chai-http";
+import { describe, it } from 'mocha';
+import app from "../app";
+import userMock from './mock/userMocks'
+
+chai.use(chaiHttp);
+
+const busId = 2
+describe(`/GET - Enquire Bus's Info`, () => {
+
+    it('should return an object', (done) => {
+        chai.request(app)
+            .get('/api/v1/buses/info/' + busId)
+            .end((err, res) => {
+                expect(res.statusCode).to.equal(200)
+                expect(res.body).to.be.an("object")
+                done()
```

Comment:
Inspect the returned object to see the inner content.

Reply:
A better name would have been 'buses`, this might confusion when calling the API in the frontend but we can leave it for now.